dev/google-drive-files/file-ids.tsv

```diff
@@ -0,0 +1,45 @@
+1rf8L-TygYD0FyNEFpG5yzvrz0aRzQWa_	STEP (assembly)/OpenArm_v1.1_leader.STEP
+1KxRKvIAmEH6EU9x1paPJFVpmgKq8uA98	STEP (assembly)/OpenArm_v1.1_follower.STEP
+1Isd2kOAkzInpuACZoU4g6_Pvd6UkvYsS	STEP (assembly)/OpenArm_v1.0_leader.step
+101j23u_6W74ipn4SQLaqvfbzI7kh5W3v	STEP (assembly)/OpenArm_v1.0_follower.step
```

Comment:
Could you remove `v1.0` data?

Reply:
I manually removed it for now.

What kind of exclusion logic should be added to the script?
Specifying the next version and excluding all others?